README.md

```diff
@@ -17,47 +17,47 @@ Our chat channel is on gitter : https://gitter.im/fossasia/susper.com
 
 ## Components and Technology
 
-This is a front-end for Susper running on Yacy server. The retrieval of search results is done using YaCy search API.
+This is a front-end for Susper running on YaCy server. The retrieval of search results is done using YaCy search API.
 
 * Solr and Javascript(JSON)
 
-Search results are displayed using Solr server which is embedded into YaCy. All search results must be provided by a YaCy search server which includes a Solr with a specialized JSON result writer. When a search request is made in one of the search templates, a http request is made to YaCy. The response is done in JSON because that can much better be parsed than XML in JavaScript. To overcome the same orgin policy in JavaScript, the result is capsuled into a JSONP response.
+Search results are displayed using Solr server which is embedded into YaCy. All search results must be provided by a YaCy search server which includes a Solr with a specialized JSON result writer. When a search request is made in one of the search templates, an HTTP request is made to YaCy. The response is done in JSON because that can much better be parsed than XML in JavaScript. To overcome the same orgin policy in JavaScript, the result is capsuled into a JSONP response.
```

Comment:
an HTTP -> a HTTP

Reply:
Sorry, but the correct form is "An HTTP".

Reply:
Yes! @ValentinBercaru is right.

HTTP is pronounced starting with a vowel sound (eh). So, `an` will be used.